src/extension_commands.ml

```diff
@@ -131,6 +133,124 @@ let _open_current_dune_file =
   in
   command Extension_consts.Commands.open_current_dune_file handler
 
+let _jump_to_prev_hole, _jump_to_next_hole =
+  let pick_next_hole current_pos ~sorted_non_empty_holes_lst:holes =
```

Comment:
You can make this `~sorted_non_empty_holes_list`.

Omitting a single letter doesn't change much.